cardano/base/templatetags/tags.py

```diff
@@ -13,3 +13,13 @@ def lookup_country_vouched(countries, key):
 def lookup_country_total(countries, key):
     q = max(countries, key=lambda d: (d['name'] == key))
     return q['total_diff']
+
+
+@register.filter
+def diff(number):
```

Comment:
Please change the ``def diff`` name to something more descriptive eg. ``glyphicon_type`` or something (i am not sure about the context, but diff is kinda misleading).